tket2/src/passes/chunks.rs

```diff
@@ -155,7 +155,7 @@ impl Chunk {
                 ConnectionTarget::TransitiveConnection(input_connection)
             } else {
                 // Translate the original chunk node into the inserted node.
-                (*node_map.get(&node).unwrap(), port).into()
+                ConnectionTarget::InsertedOutput(*node_map.get(&node).unwrap(), port)
```

Comment:
You could leave this line unchanged, right, as you are `#[derive(...,From)]` on ConnectionTarget and `#[from]` on Inserted(In,Out)put? I'd either use the From derivations or not derive them

Reply:
dropped the From, the explicit version is more clear